Okta.Xamarin/Okta.Xamarin/IOidcClient.cs

```diff
@@ -111,7 +111,7 @@ public interface IOidcClient
         /// <param name="token">The token to introspect.</param>
         /// <param name="authorizationServerId">The authorization server ID.</param>
         /// <returns>Dictionary{string, object}.</returns>
-        Task<Dictionary<string, object>> IntrospectAsync(TokenKind tokenKind, string token, string authorizationServerId = "default");
+        Task<Dictionary<string, object>> IntrospectAsync(TokenKind tokenKind, string token, string authorizationServerId = null);
```

Comment:
I suggest you move the authorization server ID to the client constructor. Once you create the client, you can use the same AS in all your API calls.

Reply:
I'll double check with the users to see how they feel about this suggestion; I'm fine to take this approach but I want to validate that the paradigm is in line with expected usage.

The reason for the change described here: https://github.com/okta/okta-oidc-xamarin/issues/58.

Reply:
Looking more closely, I think your suggestion makes sense.  I've asked for additional input from one of our consuming developers.

Reply:
Confirmed with outside developer that this is a viable approach.